package-lock.json

```diff
@@ -22950,6 +23262,24 @@
         "node": ">=16.20.1"
       }
     },
+    "node_modules/mongodb-cloud-info": {
+      "version": "2.1.3",
+      "resolved": "https://registry.npmjs.org/mongodb-cloud-info/-/mongodb-cloud-info-2.1.3.tgz",
+      "integrity": "sha512-SxvjPZ+4CsPhe0Now9aLpiSHmQGwm1jYu5hshBMkq3hAKnLrBuOwK/3TovY4SQl6h6Utqkx2rBHqLbO9incEZA==",
+      "dependencies": {
+        "cross-fetch": "^3.1.6",
+        "gce-ips": "^1.0.2",
```

Comment:
This isn't being used as a production dependency of `mongodb-cloud-info` and adds a decent number of extra dependencies, let's make `gce-ips` a dev dependency in `mongodb-cloud-info` first before we start pulling them in?